### PR TITLE
Fixes #5397 missing initialisation

### LINKED ISSUE #5397: Initialisation are missing in models/event_orga.py
**Describe the bug**

in models/event_orga.py, initialisation of `starts_at` and `payment_currency` is missing

**To Reproduce**

**Expected behavior**

**Stacktrace**

**Additional details (please complete the following information):**
 - OS: [e.g. MacOS, Ubuntu, CentOS]
 - Python Version [e.g. `3.5`, `3.6`]
 - `HEAD` Commit hash [e.g. `4629c62`]

**Additional context**

I'll be filing a PR for this. Please review it :smiley:

### PR DESCRIPTION
**Pull request by prabhupant**
Fixes #5397 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
fixes the missing initialisations in event_orga.py

#### Changes proposed in this pull request:

- added the default values of those in init() function
-
-

### CODE REVIEW

File: app/models/event_orga.py [hunk 1]
```diff
@@ -13,9 +13,13 @@ class EventOrgaModel(SoftDeletionModel):
     payment_currency = db.Column(db.String, nullable=False)
 
     def __init__(self,
-                 name=None):
+                 name=None,
+                 starts_at=None
+                 payment_currency=None):
```
[the remark this answers is not in the document]
Comment (prabhupant) [reply app/models/event_orga.py]: resolved it in the latest commit to this branch

File: app/models/event_orga.py [hunk 2]
```diff
@@ -13,9 +13,13 @@ class EventOrgaModel(SoftDeletionModel):
     payment_currency = db.Column(db.String, nullable=False)
 
     def __init__(self,
-                 name=None):
+                 name=None,
+                 starts_at=None,
+                 payment_currency=None):
 
         self.name = name
+        self.starts_at = datetime.utcnow()
```
[the remark this answers is not in the document]
Comment (prabhupant) [reply app/models/event_orga.py]: resolved it in the latest commit to this branch

File: app/models/event_orga.py [hunk 3]
```diff
@@ -13,9 +15,13 @@ class EventOrgaModel(SoftDeletionModel):
...
```
Comment (iamareebjamal) [inline app/models/event_orga.py]: If you are setting this here, then adding a constructor parameter is useless

**Comment by bhaveshAn** [comment]
Please squash the commits

File: app/models/event_orga.py (hunk 3 above)
Comment (poush) [reply app/models/event_orga.py]: Also, I think adding here makes more sense.

**Comment by prabhupant** [comment]
I need help on squashing the commits

**Comment by iamareebjamal** [comment]
First fix your branch:

```
git pull upstream development --rebase
git push origin 5397-fixes-initialisation -f
```

**Comment by CosmicCoder96** [comment]
@prabhupant Update the PR as instructed, please.

**Comment by prabhupant** [comment]
@CosmicCoder96 i need help on rebasing...i tried rebasing but i did something wrong, not sure what. please help

**Comment by CosmicCoder96** [comment]
@prabhupant can you describe what you did and what went wrong ?

**Comment by prabhupant** [comment]
@CosmicCoder96 i was trying to rebase it according to the usual way, but something messed up. The branch had 7 commits initially but after rebasing i ended up with 14 commits :sweat_smile: . Should i delete this branch and PR and create a new branch and file a new PR?

**Comment by CosmicCoder96** [comment]
@prabhupant 
> Should i delete this branch and PR and create a new branch and file a new PR?
Yes, that would be great, be sure to read the best practices, so you don't repeat what I have already corrected on your other PRs :)

**Comment by prabhupant** [comment]
@CosmicCoder96 on it :+1:

**Pull request closed without merging**